examples/deadline/EC2-Image-Builder/components/deadline-windows.component.template

```diff
@@ -0,0 +1,60 @@
+name: Deadline-${version}
+schemaVersion: 1.0
+
+phases:
+  -
+    name: 'build'
+    steps:
+      -
+        name: DownloadDeadlineClient
+        action: S3Download
+        timeoutSeconds: 120
+        onFailure: Abort
+        maxAttempts: 3
+        inputs:
+          -
+            source: '${s3uri}'
+            destination: 'C:\DeadlineClient-${version}-windows-installer.exe'
+      -
+        name: InstallDeadline
+        action: ExecutePowerShell
+        timeoutSeconds: 600
+        onFailure: Abort
+        maxAttempts: 1
+        inputs:
+          commands:
+            - '$argList = "--mode unattended --licensemode UsageBased  --connectiontype Remote --noguimode true --slavestartup false --restartstalled true --launcherservice true --serviceuser `"NT AUTHORITY\NetworkService`" --autoupdateoverride false"'
+            - 'Start-Process -FilePath {{ build.DownloadDeadlineClient.inputs[0].destination }} -ArgumentList $argList -Wait'
+      -
+        name: ConfigureAdmin
+        action: ExecutePowerShell
+        timeoutSeconds: 600
+        onFailure: Abort
+        maxAttempts: 1
+        inputs:
+          commands:
+            - 'if (-not (net localgroup administrators | Select-String "^NT AUTHORITY\\NETWORK SERVICE$" -Quiet)) {
+                net localgroup administrators /add "NT AUTHORITY\NETWORK SERVICE"
+              }'
+      -
+        name: ConfigureDeadlineLaunch
+        action: ExecutePowerShell
+        timeoutSeconds: 600
+        onFailure: Abort
+        maxAttempts: 1
+        inputs:
+          commands:
+            - 'New-ItemProperty -Path "HKLM:\System\CurrentControlSet\Control" -Name "ServicesPipeTimeout" -Value 60000 -PropertyType DWORD -Force | Out-Null'
+            - '(Get-Content C:\ProgramData\Amazon\EC2-Windows\Launch\Config\LaunchConfig.json).replace("`"setComputerName`": false", "`"setComputerName`": true") | Set-Content C:\ProgramData\Amazon\EC2-Windows\Launch\Config\LaunchConfig.json'
+            - 'C:\ProgramData\Amazon\EC2-Windows\Launch\Scripts\InitializeInstance.ps1 -Schedule'
+            - 'Start-Sleep -s 50'
```

Comment:
This is how we do it when we're creating our worker AMI's that we publish. I'll add a note as to why:

> If we do not wait at this point we are running into issues with the launcher never stopping until you manually kill the Worker and launcher on the AMI.

Reply:
I spoke with the dev that originally added that sleep and he pointed out that it's no longer necessary since we are now using the `--slavestartup false` argument to prevent the worker from starting when the launcher starts. It was originally added because we were killing the launcher while the worker was still in the process of being started, which left everything in a bad state.

I've removed the sleep command and retested the template. The example was able to create an AMI and deploy it to a worker fleet without any issues.

Reply:
Sounds good. Resolving.